content/en/blog/_posts/2024-05-01-cri-streaming-explained.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: blog
+title: "Container Runtime Interface streaming explained"
+date: 2024-05-01
+slug: cri-streaming-explained
+---
+
+**Author:** Sascha Grunert
```

Comment:
If you rebase against _main_, you can put `author: Sascha Grunert` in the front matter.

Reply:
Thanks!